src/lib/localStorage.ts

```diff
@@ -0,0 +1,163 @@
+import { BackupState, Sigmoji } from "@/types";
+import { deserializeBackupState, deserializeSigmoji, serializeBackupState, serializeSigmoji } from "./helper";
+
+
+/**
+ * Checks if the storage is empty.
+ * @returns {Promise<boolean>} - A promise that resolves to a boolean indicating if the storage is empty.
+ */
+export async function isStorageEmpty(): Promise<boolean> {
+  const sigmojis = await loadSigmojis();
+  return sigmojis.length === 0;
+}
+
+/**
+ * Parses a serialized string of Sigmojis.
+ * @param {string} serializedStr - The serialized string to parse.
+ * @returns {Promise<Sigmoji[]>} - A promise that resolves to an array of Sigmojis.
+ */
+export async function parseSerializedSigmoji(
```

Comment:
This should go under helpers, or a separate serialization util as it doesn't involve local storage